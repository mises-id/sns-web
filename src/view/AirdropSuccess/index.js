/*
 * @Author: lmk
 * @Date: 2021-07-16 00:15:24
 * @LastEditTime: 2022-03-09 17:01:13
 * @LastEditors: lmk
 * @Description: createPosts page
 */
import Navbar from "@/components/NavBar";
import React from "react";
import { useTranslation } from "react-i18next";
import { Button } from "zarm";
import complete from '@/images/complete.png'
import "./index.scss";
const AirdropSuccess = () => {
  const { t } = useTranslation();
  const viewInWallet = ()=>{
    // window.open('mises-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/popup.html')
    window.mises.openRestore()
  }
  return (
    <div>
      <Navbar title={t('airdropPageTitle')} />
      <div className="m-layout m-bg-fff m-padding15 m-font18">
        <div>
          <img src={complete} width={44} height={44} alt="" className="success-img"/>
          <p className="content">{t('airdropSuccessContent')}</p>
        </div>
        <div className="viewInWallet">
          <Button block shape="round" theme="primary" onClick={viewInWallet}>
            <span className="btn-txt">{t("viewInWallet")}</span>
          </Button>
        </div>
      </div>
    </div>
  );
};
export default AirdropSuccess;
