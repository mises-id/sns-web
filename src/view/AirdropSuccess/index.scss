.content{
  font-size: 20px;
  font-weight: bold;
  color: #333;
  line-height: 25px;
  padding: 22px 30px;
  text-align: center;
}
.success-img{
  display: block;
  margin: 50px auto 0;
}
.viewInWallet{
  width: 180px;
  margin: 20px auto;
}