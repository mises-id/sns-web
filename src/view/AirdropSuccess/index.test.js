/*
 * @Author: lmk
 * @Date: 2021-07-16 10:05:53
 * @LastEditTime: 2022-03-03 16:30:29
 * @LastEditors: lmk
 * @Description: AirdropSuccess test
 */

import React from 'react';
import {shallow } from 'enzyme';
import AirdropSuccess from '.';
describe('GreatePosts', () => {
  it('should render correctly', () => {
    const component = shallow(< AirdropSuccess/>)
    expect(component).toMatchSnapshot();
  });
});