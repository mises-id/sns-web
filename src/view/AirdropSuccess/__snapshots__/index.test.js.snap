// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreatePosts should render correctly 1`] = `
<div>
  <Navbar
    title="airdropPageTitle"
  />
  <div
    className="m-layout m-bg-fff m-padding15 m-font18"
  >
    <div>
      <img
        alt=""
        className="success-img"
        height={44}
        src="complete.png"
        width={44}
      />
      <p
        className="content"
      >
        airdropSuccessContent
      </p>
    </div>
    <div
      className="viewInWallet"
    >
      <Button
        block={true}
        disabled={false}
        ghost={false}
        htmlType="button"
        loading={false}
        onClick={[Function]}
        prefixCls="za-button"
        shadow={false}
        shape="round"
        size="md"
        theme="primary"
      >
        <span
          className="btn-txt"
        >
          viewInWallet
        </span>
      </Button>
    </div>
  </div>
</div>
`;
